README.md

```diff
@@ -55,8 +55,8 @@ You can use CPLEX with JuMP via the `CPLEX.Optimizer()` solver.
 Set solver parameters using `set_parameter` from `JuMP`:
```

Comment:
Please update this `set_parameter` to `set_optimizer_attribute` too

Reply:
Changes done.